index.html

```diff
@@ -4,9 +4,44 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:wght@100;300;400&family=Roboto:wght@100;300;400&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400&family=Roboto:wght@100;300;400&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio Setup</title>
 </head>
 <body>
-    
+<header class="app-bar">
+    <a class="logo" href=" https://johntefresh.github.io/Mobile-Portfolio-setup/">My Logo</a>
```

Comment:
- [ OPTIONAL ] I have noticed your logo is still underlined, I would suggest applying the `text-decoration: none` CSS property on the logo class to remove the underline.
Am requesting this as optional because it was missed by the previous reviewer but I `highly` recommend making this change to align and match the Figma design